From: Debian MySQL Maintainers <pkg-mysql-maint@lists.alioth.debian.org>
Date: Thu, 10 Aug 2017 20:40:28 +0200
Subject: scripts__mysql_install_db.sh__no_test

## 41_scripts__mysql_install_db.sh__no_test.dpatch by  <ch@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: scripts__mysql_install_db.sh__no_test
## DP: http://bugs.mysql.com/bug.php?id=6901
---
 scripts/mysql_install_db.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/scripts/mysql_install_db.sh
+++ b/scripts/mysql_install_db.sh
@@ -414,7 +414,7 @@ then
 fi
 
 # Create database directories
-for dir in "$ldata" "$ldata/mysql" "$ldata/test"
+for dir in "$ldata" "$ldata/mysql"
 do
   if test ! -d "$dir"
   then
